Sources/MapboxMaps/Foundation/BaseMapView.swift

```diff
@@ -32,6 +30,14 @@ open class BaseMapView: UIView, CameraViewDelegate {
     /// List of completion blocks that need to be completed by the displayLink
     internal var pendingAnimatorCompletionBlocks: [PendingAnimationCompletion] = []
 
+    /// Pointer HashTable for holding camera animators
+    internal var cameraAnimatorsHashTable = NSHashTable<CameraAnimatorInterface>.weakObjects()
+
+    /// List of animators currently alive
+    public var cameraAnimators: [CameraAnimator] {
+        return cameraAnimatorsHashTable.allObjects.compactMap { $0 as? CameraAnimator }
```

Comment:
This can be simplified if we make `CameraAnimatorInterface: CameraAnimator`
```suggestion
        return cameraAnimatorsHashTable.allObjects
```

Reply:
See https://github.com/mapbox/mapbox-maps-ios/pull/282/files#r619277867

Reply:
this is done.